.github/workflows/manual-full-build.yaml

```
@@ -8,17 +8,47 @@ on:
+            container_platform: ""
+            platform_tag_suffix: ""
+            enable_binfmt: false
+          - arch: arm64
+            container_platform: linux/arm64
+            platform_tag_suffix: -linux-arm64
+            enable_binfmt: true
+          - arch: ppc64le
+            container_platform: linux/ppc64le
+            platform_tag_suffix: -linux-ppc64le
+            enable_binfmt: true
+    if: ${{ inputs.architecture == 'all' || inputs.architecture == matrix.arch }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)